app/routes/search.js

```diff
@@ -14,14 +15,16 @@ export default class SearchRoute extends Route {
   model(params) {
     const query = params.query ? params.query : '';
 
-    return this.store.query('book', {
-      page: {
-        size: ENV.BOOKS_PER_PAGE,
-      },
+    return this.infinity.model('book', {
       filter: {
         'author.name': query
       },
-      include: 'author,photos'
+      include: 'author,photos',
+      perPage: ENV.BOOKS_PER_PAGE,
+      startingPage: 1,
+      perPageParam: 'page[size]', 
+      pageParam: 'page[number]', 
+      countParam: 'meta.total_resources'
     });
```

Comment:
This looks very similar with a query made in the books, maybe we can find a way to remove duplication.